tests/framework/config/ray-cluster.mini.yaml.template

```diff
@@ -84,31 +60,10 @@ spec:
             # Change the cluster postfix if you don't have a default setting
             command: ['sh', '-c', "until nslookup $$RAY_IP.$$(cat /var/run/secrets/kubernetes.io/serviceaccount/namespace).svc.cluster.local; do echo waiting for myservice; sleep 2; done"]
           containers:
-          - name: machine-learning # must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name',  or '123-abc'
-            image: rayproject/ray:2.0.0
+          - name: ray-worker
+            image: rayproject/ray:2.1.0
             # environment variables to set in the container.Optional.
             # Refer to https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/
-            env:
-            - name: TYPE
```

Comment:
We don't have any reference on `TYPE` now?

Reply:
There are no required envs, as far as I know.